buffer/src/test/java/io/netty/buffer/PooledByteBufAllocatorTest.java

```diff
@@ -133,7 +133,7 @@ public void testWithoutUseCacheForAllThreads() {
                 /*nHeapArena=*/ 1,
                 /*nDirectArena=*/ 1,
                 /*pageSize=*/8192,
-                /*maxOrder=*/ 11,
+                /*maxOrder=*/ 9,
```

Comment:
to reduce memory overhead ?

Reply:
Yeah, I was starting to get OOMEs from this test.

Reply:
Looks like I have to change the default max order as well. Not clear why this is suddenly such a big problem.